src/Components/Cases/CaseDetail/index.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+import React from 'react';
+import CircularProgress from 'material-ui/CircularProgress';
+import { Card, CardTitle } from 'material-ui/Card';
+import type { Case } from '../types';
+
+export type Props = {
+  case: Case,
+};
+
+export class CaseDetailContainer extends React.Component {
+  props: Props;
+  render() {
+    if (!this.props.case.id) {
+      return <CircularProgress />;
```

Comment:
Would you mind center aligning this?

Looks kind of off up in the corner like that.

![circular-align](https://user-images.githubusercontent.com/22854722/32799586-7a7ed0ba-c978-11e7-9d95-c27d4276466e.gif)